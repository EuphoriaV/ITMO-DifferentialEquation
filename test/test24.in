3.3
(8.25 + (((x - 7.08) + 2.55) * 3.65))
4.34
5.08