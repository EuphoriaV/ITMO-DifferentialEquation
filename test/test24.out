4.6E-6